src/azure-cli/azure/cli/command_modules/hdinsight/_params.py

```diff
@@ -255,3 +277,58 @@ def load_arguments(self, _):
                        help='The name of the cluster.')
             c.argument('hosts', options_list=['--host-names'], nargs='+',
                        help='A space-delimited list of host names that need to be restarted.')
+
+        # autoscale
+        autoscale_commands = ['create', 'update', 'delete', 'show']
+        for command in autoscale_commands:
+            with self.argument_context('hdinsight autoscale ' + command) as c:
+                c.argument('cluster_name', options_list=['--cluster-name'],
+                           completer=get_resource_name_completion_list('Microsoft.HDInsight/clusters'),
+                           help='The name of the cluster.')
+
+        for command in ['create', 'update']:
+            with self.argument_context('hdinsight autoscale ' + command) as c:
+                c.argument('min_workernode_count', arg_group='Load-based Autoscale',
```

Comment:
>min_workernode_count [](start = 28, length = 20)

add type=int, the same with other parameters whose value is integer

Reply:
fixed

---
In reply to: [468398388](https://github.com/Azure/azure-cli/pull/14692#discussion_r468398388) [](ancestors = 468398388)